Source/ObjectiveDropboxOfficial/Shared/Handwritten/OAuth/DBOAuthManager.h

```diff
@@ -19,7 +20,7 @@ NS_ASSUME_NONNULL_BEGIN
 ///
 /// Stores a unique identifying key for storing in `DBKeychain`.
 ///
-@interface DBAccessToken : NSObject
+@interface DBAccessToken : NSObject <NSSecureCoding>
```

Comment:
Though it may be more clear when having two classes, we'd like to minimize changes to this class to ease the migration from old to new version of SDK. This also keeps consistent to what we've done in SDKs in other languages. Right now, 2 properties are added which I think it's fine, as long as we don't overly overload the class.